### ISSUE TITLE
'terraforming ec2' throws error

### ISSUE THREAD

**Issue by erulabs** [dtan4/terraforming#103]
Hello :) I love the project!! I plan on contributing when I can find time (particularly supporting the AWS autoscaling groups).

Anyways, for most of my regions this works great. For my most populated region, I see this:

```
$ terraforming ec2
/usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/seahorse/client/plugins/raise_response_errors.rb:15:in `call': The request must contain the parameter volumes (Aws::EC2::Errors::MissingParameter)
	from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/aws-sdk-core/plugins/param_converter.rb:21:in `call'
	from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/aws-sdk-core/plugins/response_paging.rb:26:in `call'
	from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/seahorse/client/plugins/response_target.rb:21:in `call'
	from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/seahorse/client/request.rb:70:in `send_request'
	from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.1.10/lib/seahorse/client/base.rb:207:in `block (2 levels) in define_operation_methods'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/resource/ec2.rb:73:in `block_devices_of'
	from (erb):18:in `block in apply_template'
	from (erb):1:in `each'
	from (erb):1:in `apply_template'
	from /usr/local/Cellar/ruby/2.2.2/lib/ruby/2.2.0/erb.rb:863:in `eval'
	from /usr/local/Cellar/ruby/2.2.2/lib/ruby/2.2.0/erb.rb:863:in `result'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/util.rb:4:in `apply_template'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/resource/ec2.rb:19:in `tf'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/resource/ec2.rb:7:in `tf'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/cli.rb:132:in `execute'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/lib/terraforming/cli.rb:23:in `ec2'
	from /usr/local/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /usr/local/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /usr/local/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /usr/local/lib/ruby/gems/2.2.0/gems/terraforming-0.1.2/bin/terraforming:5:in `<top (required)>'
	from /usr/local/bin/terraforming:23:in `load'
	from /usr/local/bin/terraforming:23:in `<main>'
```

I'm looking into this, and believe it has something to do with an attached EBS volume. Let me know if you have any thoughts!

Thanks!

**Comment by dtan4** [comment]
> I love the project!!

Thank you!!!!!! :heart: 

>  I plan on contributing when I can find time (particularly supporting the AWS autoscaling groups).

Thanks! I always welcome your pull request :octocat: 

> I'm looking into this, and believe it has something to do with an attached EBS volume. Let me know if you have any thoughts!

Which was the status of this instance _runnning_ or _terminated_? 
I met this error when terraforming _terminated_ instance, so I implemented not to terraforming terminated instances at #102 .
If you are the same, please try v0.1.2 including this fix.

(In either case, I need to implement a guard before the API calling...)

**Comment by dtan4** [comment]
@erulabs I added a fix not to raise this exception, and just released v0.1.3 including this fix now :shipit: 
Please try v0.1.3 with your environment. If there is no problem, please close this issue :closed_book:

**Comment by dtan4** [comment]
If this problem is reproduced, please reopen this issue. Thanks!
